packages/api-core/src/marketplace-products/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './fetchers';
```

Comment:
Can we also rename the existing `site-marketplace` directory to `site-marketplace-products` for consistency then? Is it a good idea?

Reply:
I think it would make it more consistent, yes. Wdyt @sirbrillig?

Reply:
Sorry, I was AFK. I think whatever you feel makes the most sense is good. 👍